MediaBrowser.Model/FileOrganization/EpisodeFileOrganizationRequest.cs

```diff
@@ -13,5 +15,38 @@ public class EpisodeFileOrganizationRequest
         public int? EndingEpisodeNumber { get; set; }
 
         public bool RememberCorrection { get; set; }
+        public string NewSeriesName { get; set; }
+
+        public string NewSeriesYear { get; set; }
+
+        public string NewSeriesProviderIds { get; set; }
+
+        public string TargetFolder { get; set; }
+
+        public Dictionary<string, string> NewSeriesProviderIdsDictionary
```

Comment:
This is a little bit messy. The model object should be kept clean and have only the strongly typed dictionary. Leave the parsing up at the api level. I don't even think you need that anyway though, just a Dictionary property.